_posts/2023-04-09-haberdashers-win-sr2023.md

```diff
@@ -0,0 +1,110 @@
+
+## Knockouts
+
+The knockout rounds saw teams from the league matches pitted against one another in intense, high-stakes battles. With each round, the competition grew fiercer, and the robots were pushed to their limits as they raced to collect tokens and secure their place in the finals.
+
+## The Final
+
+{% include figure.html class="left" src="/images/content/blog/sr2023/final.JPG"
+           caption="Beginning of the SR2023 Final" alt="Three competitors placing their robots into the arena with a crowd of onlookers watching" %}
+
+In the nail-biting final, Haberdashers’ School faced off against Abingdon School, Gymnasium Markt Indersdorf, and Harris Westminster Sixth Form. Harris Westminster Sixth Form managed to successfully grab Haberdashers’ gold token, but were not able to get it solely back to their own scoring zone. Whilst this was happening, Gymnasium Markt Indersdorf tried to grab a gold token with their arms, but missed slightly and instead grabbed the Abingdon School robot, slowly pulling on the robot and some of its wires for the rest of the match! At the same time, the Haberdashers’ robot slowly but steadily collected bronze and silver tokens and deposited them in their zone, before going back and getting more. The teams displayed exceptional skill and teamwork, and after a fierce battle, Haberdashers’ School emerged victorious, claiming the first-place title in Student Robotics 2023.
+
+## Results
+
+- Winner: HAB – Haberdashers’ School
+- 2nd place: ABS – Abingdon School
+- 3rd place: MAI – Gymnasium Markt Indersdorf
+- Committee award: QMC – [Queen Mary’s College](https://www.qmc.ac.uk/). Award sponsored by University of Southampton ECS
+- Rookie award: SHB – [Sherborne School](https://www.sherborne.org/). Award sponsored by ITDev
+- Robot and team image award: HAB – Haberdashers’ School
+- Online presence award: HAM – [Hampton](https://hamptonschool.org.uk/) & [Lady Eleanor Holles](https://www.lehs.org.uk/) Schools
+- Challenges award: QMC – Queen Mary’s College. Award sponsored by Adventurous Machines
+
+After a close match in the final, Haberdashers’ School managed to beat their opponents and win the Student Robotics 2023 competition, closely followed by Abingdon School in second and Gymnasium Markt Indersdorf in third.
+
+The Committee Award is given to the team that displays the most extraordinary ingenuity in the design of their robot. As engineers, we appreciate elegance, simplicity, and robust engineering. This year, the committee award was given to Queen Mary’s College for their consistent excellence in all their matches. Their robot reliably and repeatedly executed their strategy to great success.
```

Comment:
Suggestion: Given the award is "extraordinary ingenuity in the *design* of their robot", perhaps we should add a photo of QMC's robot, too?

Reply:
Ingenuity in design doesn't necessarily mean interesting to look at. I don't think we need to include a picture but if there is one you want to include send me a link then I can add it :)